src/services/datahub-client.doubles.ts

```diff
@@ -191,14 +191,16 @@ export class HardcodedDatahubClient implements DatahubClient {
     //};
   }
 
-  public async getDailyVisitsByPeriod({
+  public async getDailyAndHoursVisitsByPeriod({
     domainName,
     dateFrom,
+    periodByhours,
   }: {
     domainName: string;
     dateFrom: Date;
-  }): Promise<DailyVisitsResult> {
-    console.log('getDailyVisitsByPeriod', { domainName, dateFrom });
+    periodByhours: boolean;
```

Comment:
In general, using boolean parameters is not a good idea, because they are not so descriptive and they does not scale well. For example if `period` is not hour, is it day? is it month? week?.

Maybe you can use two public methods calling a common private one.

Reply:
Also, Relay API has a nice way to deal with that, I cannot find it now, but we can talk it later.